src/app/(proper_react)/(redesign)/(public)/PlansTable.tsx

```diff
@@ -88,6 +90,31 @@ export const PlansTable = (props: Props & ScanLimitProp) => {
     currencyDisplay: "narrowSymbol",
   });
   const [billingPeriod, setBillingPeriod] = useState<BillingPeriod>("yearly");
+  const searchParam = useRef(new URLSearchParams());
+
+  useEffect(() => {
```

Comment:
I don't think this needs a ref and an effect? See also https://react.dev/learn/you-might-not-need-an-effect.

Reply:
Good catch! I forgot to take this out thanks